gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -76,26 +76,30 @@
  * <pre>
  * Gson gson = new Gson(); // Or use new GsonBuilder().create();
  * MyType target = new MyType();
- * String json = gson.toJson(target); // serializes target to Json
+ * String json = gson.toJson(target); // serializes target to JSON
  * MyType target2 = gson.fromJson(json, MyType.class); // deserializes json into target2
  * </pre>
  *
  * <p>If the object that your are serializing/deserializing is a {@code ParameterizedType}
  * (i.e. contains at least one type parameter and may be an array) then you must use the
- * {@link #toJson(Object, Type)} or {@link #fromJson(String, Type)} method. Here is an
- * example for serializing and deserializing a {@code ParameterizedType}:
- *
+ * {@link #toJson(Object, Type)} or {@link #fromJson(String, TypeToken)} method. Gson provides
```

Comment:
Since the text says "you _must_ use", I think it should detail all the options, so both `Type` and `TypeToken` overloads. There may be cases where it is still preferable to use the `Type` one. `TypeToken` could come first, though, and the example show just that.

Reply:
I have reworded the whole paragraph a bit because it was also incorrect regarding serialization. I hope the new wording is fine; though please let me know if I should adjust something.